### PR TITLE
Fix the navigations

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Main nav should consist of: 

Products
Services
Clients
Defacto
Blog
Vacatures 

But blog and vacatures should render in the Defacto group on mobile and in the footer, so introducing 'not_main' for these two items that should not render there.

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
@sn3p can you review please?

**Comment by RonaldDijkstra** [comment]
https://deploy-preview-2034--defacto-nl.netlify.com/

File: data/navigations_de.yml [hunk 1]
```diff
@@ -93,6 +94,11 @@
     - title: Jobs
       url: /jobs/index.html
       icon: icons/jobs.svg
+      not_main: true
```
Comment (sn3p) [inline data/navigations_de.yml]: I would go for:

```suggestion
      main: false
```

So defaults to true unless set the false.

File: source/partials/header/_nav-dropdowns.erb [hunk 2]
```diff
@@ -43,11 +43,13 @@
                 <div class="link-group">
 
                   <% item.links.each do |link| %>
-                    <% nav_link_to(link.url, class: "link-container with-icon") do %>
-                      <span class="link-title link-icon">
-                        <%= inline_svg link.icon, title: link.title %>
-                        <%= link.title %>
-                      </span>
+                    <% if link.not_main == nil %>
```
Comment (sn3p) [inline source/partials/header/_nav-dropdowns.erb]: Then you can do:

```suggestion
                    <% unless link.main == false %>
```

Comment (RonaldDijkstra) [reply source/partials/header/_nav-dropdowns.erb]: Cool. Thanks. It's fixed 👍

**Pull request merged**